maps-compose/src/main/java/com/google/maps/android/compose/CameraPositionState.kt

```diff
@@ -235,17 +239,23 @@ class CameraPositionState(
     private fun performAnimateCameraLocked(
         map: GoogleMap,
         update: CameraUpdate,
+        durationMillis: Int,
         continuation: CancellableContinuation<Unit>
     ) {
-        map.animateCamera(update, object : GoogleMap.CancelableCallback {
+        val cancelableCallback = object : GoogleMap.CancelableCallback {
             override fun onCancel() {
                 continuation.resumeWithException(CancellationException("Animation cancelled"))
             }
 
             override fun onFinish() {
                 continuation.resume(Unit)
             }
-        })
+        }
+        if (durationMillis < 0) {
```

Comment:
See note above about `0` not being a valid value.

Reply:
Updated this in [09c990e](https://github.com/googlemaps/android-maps-compose/pull/83/commits/09c990e8b3aaa4f510f236e06aca36770647ca80) to check against null